core/src/main/java/org/polypheny/db/sql/ddl/SqlDdlNodes.java

```diff
@@ -60,8 +60,8 @@ private SqlDdlNodes() {
     /**
      * Creates a CREATE SCHEMA.
      */
-    public static SqlCreateSchema createSchema( SqlParserPos pos, boolean replace, boolean ifNotExists, SqlIdentifier name ) {
-        return new SqlCreateSchema( pos, replace, ifNotExists, name, SchemaType.RELATIONAL );
+    public static SqlCreateSchema createSchema( SqlParserPos pos, boolean replace, boolean ifNotExists, SqlIdentifier name, SchemaType schemaType ) {
```

Comment:
Is it possible to create non-relational schemas using PolySQL?

Reply:
Yes, this is one of the only extensions of the PolySQL of this PR. 
This is handy to have a way of using the same language, for example from the frontend, to create both types of possible schemas.
But I am open to change it, if it could lead to confusion in your opition?